front/app/components/FormBuilder/components/FormFields/index.tsx

```diff
@@ -162,16 +163,19 @@ const FormFields = ({
         </Box>
         {/* Only relevant for survey */}
         {lastPage.key === 'survey_end' && (
-          <Box mt="40px">
-            <FormField
-              field={lastPage}
-              selectedFieldId={selectedFieldId}
-              onEditField={onEditField}
-              builderConfig={builderConfig}
-              fieldNumbers={fieldNumbers}
-              closeSettings={closeSettings}
-            />
-          </Box>
+          <>
+            {userFieldsInFormNotice && userFieldsInFormNotice()}
+            <Box mt={userFieldsInFormNotice ? '0' : '40px'}>
+              <FormField
```

Comment:
Rather than putting the mt here, you could also put it in the component return by userFieldsInFormNotice() ? :thinking: Up to you though :) 

`              <UserFieldsInFormNotice
                projectId={projectId}
                phaseId={phase.data.id}
              />`